src/components/model-form/model-form.js

```diff
@@ -27,13 +27,19 @@ import CheckboxOptions from '../checkbox-options';
 import ImageUploadContainer from '../../containers/image-upload-container';
 import Editor from '../editor';
 import useBusinessHandlers from '../../utils/use-business-handlers';
+import {
+  setSnackBarMessage,
+  setSnackBarSeverity,
+  setSnackBarStatus
+} from '../../redux/snackbar/snackbar.actions';
 
 const {
   MODEL_VALIDATION_ERROR,
-  MODEL_ERROR_MESSAGE
+  MODEL_ERROR_MESSAGE,
+  NO_PHOTO
```

Comment:
It seems better to rename it from NO_RHOTO to PHOTO_NOT_PROVIDED

Reply:
Solved